core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/ApiServiceSpec.scala

```diff
@@ -417,6 +405,19 @@ trait ApiServiceSpec
                                   samDAO
       )
 
+    val spendReportingBigQueryService = bigQueryServiceFactory.getServiceFromJson("json", GoogleProject("test-project"))
```

Comment:
This is just moved after the creation of `workspaceServiceConstructor` so I can pass it in as a parameter